en/render/material.md

```diff
@@ -0,0 +1,93 @@
+# Material
+
+Material can be used to control the visual effects of Renderer components in the scene. In short, material is used to specify the properties of an object's surface, such as color, shininess, emissive, opacity and so on.
+
+![Material](./material/material.png)
+
+## Material Properties
+
+| Properties | Explanation	             |
+| ---------- | ---------------- |
+| Effect     | All effect resources in the current project are listed in the Effect drop-down box, and you can select the effect resource used by the current material. And the other properties will be updated when the Effect is switched. |
+| Technique  | All Techniques in the currently used Effect are listed in the Technique drop-down box.<br>There may be more than one Technique in an Effect, and each Technique may be applicable to different situations. Such as Technique, which is less effective but has better performance, and more suitable for mobile platforms.<br>The Pass list will be updated simultaneously when the Technique is switched. |
+| Pass       | The Pass list lists all Passes in the currently used Technique.<br>Each Pass may have different properties and definitions that you can set separately. If the property is wrapped by a definition, you need to check the definition to see the corresponding property.
+
+## Create and use material
+
+Material can be created and used in both the editor and the code.
+
+### Create and use in the editor
+
+#### Create material
+
+Just click the **+** button at the top left of the **Assets**, and then select the **Material**.
+
+![](./material/create-material.png)
+
+Another way is to select the folder in the **Assets** panel where you want to store the materials, then right-click and select **Create -> Material**.
+
+#### Use material
+
+1. After creating the Material, select the New Material in **Assets** panel, and then set the properties in the **Properties** panel. After editing, click the **Apply** button on the top right.
+
+2. Then select the node where the Renderer component is located in the **Node Tree** panel, and drag the New Material to the **Materials** property box of the Renderer component.
+
+    ![](./material/set-material.png)
+
+### Create and use in code
+
+The Material created in code are divided into **Shared Material** and **Material Variant**. If you change the Shared Material, it will affect the Material Variant synchronously.
```

Comment:
`Materials created in runtime can be divided into **Shared Material** and **Material Variant** types. If you change a Shared Material, it will affect all Material Variants synchronously.`